data_store/transformation/towns_fund/tf_transform_r3.py

```diff
@@ -343,7 +356,7 @@ def extract_project_progress(df_data: pd.DataFrame, project_lookup: dict, report
     """
     # if round 4 or 5, ingest two additional columns
     df_data = df_data.iloc[18:39, 2:15] if reporting_round in (4, 5) else df_data.iloc[18:39, 2:13]
-    df_data = df_data.rename(columns=df_data.iloc[0]).iloc[1:]
+    df_data = df_data.rename(columns=df_data.iloc[0].to_dict()).iloc[1:]
```

Comment:
👀

Reply:
`to_dict` made mypy happy. It's a small price to pay to make `mypy` happy.

It takes something that looks like this:
```
2     B1\nTD 5% CDEL Pre-Payment; RDEL Capacity Fund...
4     TD 5% CDEL Pre-Payment\n(Towns Fund FAQs p.46 ...
5                              TD RDEL Capacity Funding
8                                TD Accelerated Funding
12                                       Guidance Notes
Name: 13, dtype: object
```
Converts it to a dict:
```
{
    2: 'B1\nTD 5% CDEL Pre-Payment; RDEL Capacity Funding; and Accelerated Funding',
    4: 'TD 5% CDEL Pre-Payment\n(Towns Fund FAQs p.46 - 49)',
    5: 'TD RDEL Capacity Funding',
    8: 'TD Accelerated Funding',
    12: 'Guidance Notes'
}
```

The result is still the same. Converts the 1st row of a DataFrame to it's header.
Before (note that header is [2, 12...]):
```
                                                   2   ...                                                 12
13  B1\nTD 5% CDEL Pre-Payment; RDEL Capacity Fund...  ...                                     Guidance Notes
14  Beyond these three funding types, have you rec...  ...                                                NaN
15  Please indicate how much of your allocation ha...  ...                                                NaN
16  Please confirm whether the amount utilised rep...  ...                                                NaN
17  Please describe when funding was utilised and,...  ...                                                NaN
18  Please select the option that best describes h...  ...  Given your answer on the 5% CDEL Pre-Payment, ...
19  Please explain in detail how the funding has, ...  ...                                                NaN

[7 rows x 5 columns]
```
After:
```
   B1\nTD 5% CDEL Pre-Payment; RDEL Capacity Funding; and Accelerated Funding  ...                                     Guidance Notes
14  Beyond these three funding types, have you rec...                          ...                                                NaN
15  Please indicate how much of your allocation ha...                          ...                                                NaN
16  Please confirm whether the amount utilised rep...                          ...                                                NaN
17  Please describe when funding was utilised and,...                          ...                                                NaN
18  Please select the option that best describes h...                          ...  Given your answer on the 5% CDEL Pre-Payment, ...
19  Please explain in detail how the funding has, ...                          ...                                                NaN

[6 rows x 5 columns]
```

This is from line 461 where we do this. It's easier to see the change.